pkg/kubelet/cm/devicemanager/manager_test.go

```diff
@@ -1072,9 +1072,9 @@ func TestPodContainerDeviceAllocation(t *testing.T) {
 		if testCase.expectedContainerOptsLen == nil {
 			as.Nil(runContainerOpts)
 		} else {
-			as.Equal(len(runContainerOpts.Devices), testCase.expectedContainerOptsLen[0])
-			as.Equal(len(runContainerOpts.Mounts), testCase.expectedContainerOptsLen[1])
-			as.Equal(len(runContainerOpts.Envs), testCase.expectedContainerOptsLen[2])
+			as.Len(runContainerOpts.Devices, testCase.expectedContainerOptsLen[0])
```

Comment:
/cc @matthyx. The order of expect and actual in the original as.Equal function is reversed. When the test case reports an error, the error message is confusing.